### PR TITLE
use fortio

### PR DESCRIPTION
**Pull request by ldemailly**
use [fortio](https://github.com/istio/fortio/) (istio's perf tool) rather than a custom wrk2 image

### CODE REVIEW

File: exercise-8/README.md [hunk 1]
```diff
@@ -10,12 +10,11 @@ With simple shell script:
...
 ```
 
-Or, with wrk2:
+Or, with fortio:
 
 ```sh
-docker pull saturnism/wrk2
-docker run -ti --rm saturnism/wrk2 \
-  -d 5s -R 5 http://$INGRESS_IP/hello/world
+docker run istio/fortio load -t 5m -qps 5 \
```
Comment (ldemailly) [inline exercise-8/README.md]: changed from 5s in original to 5 minutes as it seems you'd want more than 5 seconds of load

it also seems simpler to just docker run than pull + run

**Comment by ldemailly** [comment]
fortio's image is 3mb compressed vs 5mb for the original

**Comment by retroryan** [comment]
Thank you @ldemailly !  fortio is a much better way of load testing!

**Comment by ldemailly** [comment]
Thanks!

**Pull request merged**